### ISSUE TITLE
Exclude Weekends?

### ISSUE THREAD

**Issue by gramanero** [jeffpriz/devops-pr-stats#28]
I'm wondering if having weekends included in the total time for a PR is skewing the PR Average Open Time since I would think most organizations do not require their engineers to work on the weekends. Opening a PR on a Friday and not having it close until Monday may actually only be a 24-hour period if it were not for crossing over Saturday and Sunday. Or perhaps allow the ability to exclude weekends so that each can choose their own way that the average is being calculated?

**Comment by seawerst** [comment]
This is a great idea. @jeffpriz are you still open for PR? I see one from a couple of month ago that is still waiting.

**Comment by jeffpriz** [comment]
yeah, would definitely take a PR for that. I would see the ability to check/uncheck the exclusion of weekends as something I'd want out it.
